src/app/accessibility/index.njk

```diff
@@ -0,0 +1,120 @@
+{% extends "base-template.njk" %}
+
+{% block pageTitle %}
+  GAR | Accessibility Statement
+{% endblock %}
+
+{% block head %}
+  {% include "head.njk" %}
+{% endblock %}
+
+{% block header %}
+  {% include "header-pre-user.njk" %}
+{% endblock %}
+
+{% block beforeContent %}
+  {% include "phase.njk" %}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+	<div class="govuk-grid-column-two-thirds">
+  <div class="govuk-notification-banner govuk-notification-banner--success" id="preference_changed" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner" tabindex="-1" hidden>
+  <div class="govuk-notification-banner__header">
+    <h2 class="govuk-notification-banner__title" id="govuk-notification-banner-title">
+      Success
+    </h2>
+  </div>
+  <div class="govuk-notification-banner__content">
+    <p class="govuk-notification-banner__heading">
+      You’ve set your cookie preferences. <a class="govuk-notification-banner__link" href="javascript:history.back()">Go back to the page you were looking at</a>.
```

Comment:
Cookie stuff is already in the `base-template.njk`.
*Should the banner be defined here or not?

Reply:
That is correct, I take the wrong file  for reference. Updated the code